charts/redis-ha/values.yaml

```diff
@@ -0,0 +1,102 @@
+## Configure resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+##
+
+image: redis
+tag: 4.0.11-stretch
+## replicas number for each component
+replicas: 3
+
+## Redis specific configuration options
+redis:
+  port: 6379
+  masterGroupName: mymaster
```

Comment:
I think it's ok if we switch master name to `zenko`. Sentinel should allow other slaves to update their configuration and interpret this as the new master name. This should be tested obviously.

Reply:
I tested this and didn't work (seamlessly at least) because kubernetes updates the pods in reverse order. So when the redis-ha-2 goes down and comes back up, it tries to contact the master in the `zenko` group that does not exist and goes into crashloop state preventing the remaining pods from updating. However deleting all the pods manually does result in the intended group change.

Reply:
Ah damn, thanks for testing. Let's leave it as-is.